src/tests/EdFi.DataManagementService.Tests.E2E/Hooks/SetupHooks.cs

```diff
@@ -14,10 +14,9 @@ public class SetupHooks
 {
     private static IConfiguration? _configuration;
 
-    [BeforeTestRun]
+    [BeforeFeature]
```

Comment:
Starting with fresh containers before each feature. It doesn't slow the process down too much but gives us unpolluted database before each feature.